src/pubsub/floodsub.ts

```diff
@@ -12,8 +13,9 @@ export function floodsubTests (factory: DaemonFactory) {
     for (const typeB of nodeTypes) {
       runFloodsubTests(
         factory,
-        { type: typeA, pubsub: true, pubsubRouter: 'floodsub' },
-        { type: typeB, pubsub: true, pubsubRouter: 'floodsub' }
+        // RSA key ensures the `key` field is set in the generated signed message
```

Comment:
This is a workaround for a problem described here: https://github.com/libp2p/js-libp2p-pubsub/pull/118#discussion_r1091846943

Reply:
Is this needed for adding the relay test? It seems like this is for pubsub, no?

Reply:
Yes, while this change is only for pubsub, I don't want to split it into a separate PR. This is because if we use the existing pubsub tests with the new go-libp2p-daemon, they will fail. We need to use the new relay daemon since it has relay v2 enabled.

Reply:
I have a preference for creating a small PR with just these changes. But I understand the overhead. Feel free to keep the changes here.